### PR TITLE
PKG-3454

### PR DESCRIPTION
**Pull request by cbouss**
Changes:
- new feedstock

https://github.com/jupyter-widgets/ipyleaflet/tree/0.17.4

### CODE REVIEW

**Review by JeanChristopheMorinPerso — APPROVED** [review]

File: recipe/meta.yaml [hunk 1]
```diff
@@ -11,20 +11,18 @@ source:
...
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
-  build:
-    - nodejs >=16
   host:
-    - python >=3.7
+    - python
     - pip
-    - jupyter
```
Comment (M-Waszkiewicz-Anaconda) [inline recipe/meta.yaml:23]: why removed ? 
https://github.com/jupyter-widgets/ipyleaflet/blob/da6701f60d766961b2f7906bee1e1b0fea899369/pyproject.toml#L3

Comment (skupr-anaconda) [reply recipe/meta.yaml:23]: There is no `jupyter`. 
`jupyterlab` is not needed to build the package

**Review by skupr-anaconda — APPROVED** [review]
LGTM

File: recipe/meta.yaml (hunk 1 above)
Comment (cbouss) [reply recipe/meta.yaml:23]: correct

**Pull request merged**